### ISSUE TITLE
Mobile menu button issue on Kindle

### ISSUE THREAD

**Issue by Misplon** [siteorigin/siteorigin-north#105]
A user has reported a problem with opening the mobile menu using Kindle's Silk browser.

**Bug report thread:**
https://siteorigin.com/thread/3-line-menu-icon-does-not-work-in-kindles-silk-browser/#comment-94285

**Possible method for testing the problem:**
http://stackoverflow.com/questions/7703055/emulator-for-amazon-kindle

Hopefully, we can recreate the problem. Thanks @adiraomj.

**Comment by gregpriday** [comment]
The thread isn't entirely clear on what's not working? Does the entire menu not display? Does the animation just not work? We wouldn't really bother checking and fixing the animation. This might just mean that Silk doesn't support SVG animations. If the entire menu isn't displaying then we'd want to check this, but with a reasonably low priority.

**Comment by Misplon** [comment]
@gregpriday and I briefly discussed this offline. @adiraomj are you able to test on Android? If so, could you please, test the North mobile browser and see if there are issues? Thanks.

**Comment by adiraomj** [comment]
@Misplon I will check it on Android and let you know..

**Comment by adiraomj** [comment]
@Misplon I tested the theme on android using chrome. The mobile menu works as expected. I also checked out the website mentioned by the user(www.stanges.biz).

This could be kindle specific issue. After reading the support thread it does seem to be a svg animation issue, as @gregpriday has mentioned above. 

It could be helpful to know if it's the svg animation or mobile menu that's causing the problem.
